src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI.Task/GenAPITask.cs

```diff
@@ -49,6 +49,17 @@ public class GenAPITask : TaskBase
         /// </summary>
         public string[]? ExcludeAttributesFiles { get; set; }
 
+        /// <summary>
+        /// If true API Compat will not include internal attributes commonly defined to
+        /// represent language features (Eg: System.Runtime.CompilerServices.NullableAttribute, etc).
+        /// </summary>
+        public bool ExcludeInternalCompilerAttributes { get; set; }
+
+        /// <summary>
+        /// The path to one or more api inclusion files with types in DocId format.
+        /// </summary>
+        public string[]? IncludeApiFiles { get; set; }
```

Comment:
Expose these two flags in the msbuild targets file as well: https://github.com/ericstj/sdk/blob/0fee0304a78aa0b2dcddb909297556ba773e033f/src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI.Task/build/Microsoft.DotNet.GenAPI.Task.targets#L45